src/components/atoms/input/ms-input.ts

```diff
@@ -0,0 +1,104 @@
+import { LitElement, html } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+
+import { WithTheme } from '~/mixins'
+
+import { msInputStyles } from './ms-input.styles'
+
+@customElement('ms-input')
+export class MSInput extends WithTheme(LitElement) {
+  static styles = msInputStyles
+
+  @property({ type: String })
+  value = ''
+
+  @property({ type: String })
+  label = ''
+
+  @property({ type: String })
+  message = 'Informative message'
+
+  @property({ type: String })
+  successLabel = 'Success message'
```

Comment:
'error','warning','success' should be a single property like: `<ms-input status="error" `

Reply:
done!